src/html/templates/styles.css

```diff
@@ -24,65 +24,262 @@ summary::-webkit-details-marker {
+      
+      .namespaceItemTitle a {
+        @apply line-through decoration-2 decoration-stone-500/70 text-stone-500;
+      }
+    }
+    
+    .namespaceItemTitle {
+      @apply flex gap-2 items-center;
+      
+      > div:last-child {
+        @apply flex justify-between gap-4 flex-1 min-w-0 items-end overflow-hidden;
+        
+        > a {
+          @apply leading-tight truncate;
+        }
+        
+        > div {
+          @apply text-xs italic lg:mr-4 text-stone-400 truncate;
+        }
+      }
+    }
+    
+    .namespaceItemContent {
+      @apply py-1 text-sm leading-5 text-stone-600 markdown_border;
+      @apply ml-2 !important;
+    }
+  }
 }
 
-#sidepanel li > a:has(span) {
-  @apply py-1;
+.symbolGroup {
+  @apply space-y-12;
+  
+  article {
+    @apply space-y-5;
+    
+    > div:first-child {
+      @apply flex justify-between items-start;
+      
+      > div:first-child {
+        @apply font-medium space-y-1;
+      }
+    }
+  }
 }
 
-#sidepanel a:hover,
-#sidepanel a.active {
-  @apply bg-stone-200;
+.docNodeKindIcon {
+  @apply flex justify-end w-[2.125rem] shrink-0;
+  
+  div {
+    @apply rounded-full size-5 font-medium text-xs leading-5 text-center align-middle shrink-0 select-none font-mono;
+  }
+  
+  > * + * {
+    @apply -ml-1.5;
+  }
 }
 
-.compound_types > * + * {
-  @apply -ml-1.5;
+.example details {
+  summary {
+    @apply list-none flex items-center gap-2 py-2 rounded-lg w-full leading-6 cursor-pointer;
+    
+    > div:first-child {
+      @apply text-stone-600 select-none;
+    }
+  }
+  
+  &[open] summary > div:first-child {
+    @apply rotate-90;
+  }
 }
 
-.anchorable {
-  @apply relative;
+.tocable {
+  @apply flex max-lg:flex-col-reverse gap-6;
+  
+  .toc {
+    @apply flex-none max-w-52 text-sm max-lg:hidden sticky top-0 py-4 max-h-screen box-border;
+    
+    > ul {
+      @apply space-y-2 block overflow-y-auto h-full;
+      
+      ul {
+        @apply ml-4 space-y-2;
+      }
+    }
+    
+    a {
+      @apply hover:underline block overflow-x-hidden whitespace-nowrap text-ellipsis;
+    }
+  }
 }
 
-.anchorable:hover .anchor {
-  @apply block;
+.usage {
+  @apply px-6 py-5 mb-4 border max-md:border-x-0 md:rounded-lg bg-stone-50 border-stone-200;
+  
+  nav {
+    @apply flex items-center flex-row gap-2 mb-3 font-semibold;
+    
+    details {
+      > summary {
+        @apply flex gap-1 select-none cursor-pointer rounded border border-gray-300 py-2 px-3;
+      }
+      
+      > div {
+        @apply md:relative;
+        
+        > div {
+          @apply absolute max-md:inset-x-0 mt-1.5 p-2 block z-30 md:w-72 bg-white md:rounded border max-md:border-x-0 border-gray-300;
+         
+          label {
+            @apply flex items-center justify-between cursor-pointer select-none px-2 py-1 leading-normal rounded-sm hover:bg-gray-50;
+          }
+        }
+      }
+    }
+  }
+  
+  pre.highlight {
+    @apply border border-gray-300 max-md:border-x-0;
+  }
 }
 
```

Comment:
Question: is this selector nesting compiled away? 

If so, disregard this comment. But otherwise, nesting is fairly well supported, but still new enough it may be risky to use, especially without the `&` syntax. https://caniuse.com/?search=nesting

Reply:
yep, its compiled away. I know tailwind supports nesting, but as also pass the stylesheet through lightningcss